modes/tmtv/src/toolbarButtons.js

```diff
@@ -345,7 +338,6 @@ const toolbarButtons = [
         tooltip: 'PET Image Colormap',
       },
       isAction: true, // ?
-      renderer: WindowLevelMenuItem,
```

Comment:
This was causing the menu to have numbers beside it, causing it to look like it was appliable from the keyboard.